godot-core/src/log.rs

```diff
@@ -75,6 +78,7 @@ pub use crate::{godot_error, godot_print, godot_script_error, godot_warn};
 use crate::builtin::{StringName, Variant};
 use crate::sys::{self, GodotFfi};
 
+/// prints to the godot console, it's use is to be used by the godot_print! macro
```

Comment:
The wording is a little clunky here, maybe just drop "it's use is to be" and have it just say "used by the godot_print! macro".
Also it should be "its use" not "it's use".
I think you can also link to the `godot_print` macro in the doc here?
Also, missing capitalization and period.

Reply:
yes that's better, thanks I'm not a native english speaker so this is a big help.